airflow/utils/db.py

```diff
@@ -1666,11 +1667,14 @@ def upgradedb(
         _reserialize_dags(session=session)
     add_default_pool_if_not_exists(session=session)
     synchronize_log_template(session=session)
-    if _revision_greater(
-        config,
-        _REVISION_HEADS_MAP["2.9.0"],
-        _get_current_revision(session=session),
+    current_version = _get_current_revision(session=session)
```

Comment:
`current_version == _from_revision`, No?

Reply:
this was my intention for this refactor 🤦‍♂️